server/notebooks/api_views.py

```diff
@@ -45,7 +45,7 @@ class NotebookRevisionViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
 
     # revisions should be considered immutable once created
-    http_method_names = ['get', 'post', 'head']
+    http_method_names = ['get', 'post', 'head', 'delete']
```

Comment:
Could you add some unit tests to make sure this works as expected for all the possibilities (logged-in-as-owner, logged-in-as-non-owner, not-logged-in)?

Reply:
Yes, that was one of the remaining prerequisites for this PR I mentioned before. Will make sure there are tests when I circle back on this.

Reply:
@wlach as of the most recent commit I added these tests in `test_notebook_revision_api.py` - feel free to take a look and let me know if anything is amiss there.

Reply:
+1 they look great